starter/js/atm.js

```diff
@@ -1,15 +1,53 @@
 //Begin with the document ready function
+$( document ).ready(function() {
```

Comment:
You'll want the rest of the code to live *inside* of the `$( document ).ready`, just to be safe!